ginivision/src/main/java/net/gini/android/vision/review/ReviewActivity.java

```diff
@@ -291,6 +330,11 @@ public void onDocumentAnalyzed() {
     }
 
     @Override
+    public void noExtractionsFound() {
```

Comment:
The method name is a little bit confusing for me, shouldn't be something like `setNoExtractionsFound(Bool)` or `onNoExtractionsFound` (just suggestions)?